recipes-connectivity/awsclient/awsclient_1.3.1.bb

```diff
@@ -0,0 +1,35 @@
+SUMMARY = "The AWS IoT client establishes a connection to AWS IoT, executes jobs and sends status information."
+
+LICENSE = "CLOSED"
+
+SRC_URI = "\
+    https://github.com/osb-cc-esec/meta-esec-awsclient/releases/download/v${PV}/awsclient_${PV}_aarch64 \
+    file://awsclient.service \
+    file://awsclient.timer \
+"
+
+SRC_URI[md5sum] = "83b5a82ab97649fa065a3b99f5a5d8ce"
+SRC_URI[sha256sum] = "6c5183522734539c58841703ef856096a205e54f315330d3d511a6d03f1b0840"
+
+DEPENDS = "curl glib-2.0 json-glib openssl"
+
+inherit systemd
+
+SYSTEMD_AUTO_ENABLE = "enable"
+SYSTEMD_SERVICE_${PN} = "awsclient.timer"
```

Comment:
Do we have to specify the timer, service file or both here? I could not find anything in the documentation at first sight that would clarify such a case.

Reply:
We have been using it like this in the past, only the timer file has to be enabled and it will trigger the service with the same name.